projects/driver_controls_pedal/test/test_control_stalk_indicator.c

```diff
@@ -0,0 +1,16 @@
+#include "delay.h"
```

Comment:
Nothing is actually happening in this file. Let's remove it or write real tests for it. If you decide to remove it we should create a jira to make sure we don't forget about making these tests

Reply:
we decided we didnt need this indicator , forgot to remove it